Cargo.toml

```diff
@@ -100,3 +100,7 @@ unused_must_use = "deny"
 # Clippy lints are documented here: https://rust-lang.github.io/rust-clippy/master/index.html
 indexing_slicing = "warn"
 missing_safety_doc = { level = "warn" }
+pedantic = "deny"
+# Part of pedantic but not relevant to us, mostly because we sometimes erase
+# module names when re-exporting types
+module_name_repetitions = "allow"
```

Comment:
We could reexport them with an alias that includes the module-name again. 
But not a strong opinion.